pkg/osquery/tables/wifi_networks/wifi_networks.go

```diff
@@ -0,0 +1,149 @@
+package wifi_networks
```

Comment:
This should be build tagged to windows (ditto for _test.go)

Reply:
oh good catch. They were at some point in the past, but I removed it because some tools I use respect them when I don't want them to during development on my mac.